ultra/ultra/baselines/agent_spec.py

```diff
@@ -82,8 +84,6 @@ def __new__(
                 social_capacity=10,
```

Comment:
```suggestion
                social_capacity=policy_params["social_vehicles"]["social_capacity"],
```

Reply:
Also in https://github.com/huawei-noah/SMARTS/blob/46d74c415d610c565bd481489d80f93809b3aa31/ultra/ultra/baselines/agent_spec.py#L83 maybe have it reference the policy_params so it's not hard-coded? Let me know your thoughts.

Reply:
yes these were left for cleanup. as long as we have access to policy_params it should be ok to replace these

Reply:
Thank you for letting me know, the changes are implemented, please check de21070